docs_user/modules/proc_reusing-existing-subnet-ranges.adoc

```diff
@@ -1,25 +1,14 @@
 [id="reusing-existing-subnet-ranges_{context}"]
 
-= Scenario 2: Reusing existing subnet ranges
+= Reusing existing subnet ranges
```

Comment:
General comment:  I’m confused by the purpose of this module. Is any configuration required at all by the user? If not, is the purpose of this module to just explain what reusing subnet ranges means, and that no additional configuration is required?